### PR TITLE
Cumulus 1337 documentation update

### PR DESCRIPTION
**Pull request by flamingbear**
**Summary:** Documentation for setting up distribution metrics

Addresses [CUMULUS-1337: Make Metrics team handle metrics](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1337)

## Changes

* reorganized some doc files
* Add documentation for setting up distribution metrics.

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: docs/additional-deployment-options/share-s3-access-logs.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+id: share-s3-access-logs
+title: Share S3 Access Logs
+hide_title: true
+---
+
+# Sharing S3 Access Logs
+
+In the NGAP environment, the ESDIS Metrics team has set up an ELK stack to process logs from Cumulus instances.  One step is to deliver any S3 Server Access logs that Cumulus creates.  We have provided a simple node package to aid this process.
```
Comment (ifestus) [inline docs/additional-deployment-options/share-s3-access-logs.md]: I'm not quite sure what you mean by `one step`. Are there more steps to the process?

File: docs/additional-deployment-options/share-s3-access-logs.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# S3 Replicator
+
+The S3 Replicator is a node package that contains a simple lambda functions, associated permissions and the terraform instructions to replicate create-object events from one S3 bucket to another.
```
Comment (ifestus) [inline docs/additional-deployment-options/share-s3-access-logs.md]: `lambda functions` -> `lambda function`

File: docs/deployment/server_access_logging.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ hide_title: true
 
 # S3 Server Access Logging
 
-To enable [EMS Reporting](../ems_reporting.md), you need to enable [S3 server access logging][awslogging] on all protected and public buckets.
+To enable [EMS Reporting](../features/ems_reporting.md), you need to enable [S3 server access logging][awslogging] on all protected and public buckets.
```
Comment (ifestus) [inline docs/deployment/server_access_logging.md]: I like this being in a features directory.

File: docs/features/distribution-metrics.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+The metrics team has taken care of setting up logstash to ingest the files that get delivered to their bucket into their Elasticsearch instance.
+
+
+## Kibana Index
+
+Once cumulus has been configured to deliver Cloudwatch logs to the NGAP Metrics team, and you have run at least one request that has been logged, you can create a Kibana index associated with your Cumulus stack.  The metrics team has worked out a convention with the Cumulus devlopers to ensure access to your stack's logs.  The important piece is that the Kibana index is created with the name of the prefix (stackName) cumulus was deployed with.
+
+
+Before creating the Kibana index, verify that the Elasticsearch instance has been populated with at least one record[^1].  Do this by vititing the Kibana endpoint and selecting `Management` => Elasticsearch `Index Management` and type the stack's prefix into the search bar.  When you see an index with `<prefix>-cloudwatch-YYY.MM.dd` you are ready to continue.  If you don't see at least one index for your stack, check to make sure you are delivering your logs to this Elasticsearch instance and work with the metrics team to guarantee this.
```
Comment (ifestus) [inline docs/features/distribution-metrics.md]: `vititing` -> `visiting`

File: docs/additional-deployment-options/share-s3-access-logs.md (hunk 1 above)
Comment (ifestus) [reply docs/additional-deployment-options/share-s3-access-logs.md]: The first step?

Comment (flamingbear) [reply docs/additional-deployment-options/share-s3-access-logs.md]: I meant it as "one step is to deliver s3 access logs" . Another is to set up cloudwatch logs sharing.  But I'll see if I can reword.

Comment (flamingbear) [reply docs/additional-deployment-options/share-s3-access-logs.md]: Lauren took care of this in her PR.

**Pull request merged**